superset/views/base.py

```diff
@@ -142,8 +158,8 @@ def wraps(self, *args, **kwargs):
             return f(self, *args, **kwargs)
         except SupersetSecurityException as ex:
             logger.exception(ex)
-            return json_error_response(
-                utils.error_msg_from_exception(ex), status=ex.status, link=ex.link
+            return json_errors_response(
+                [ex.error], status=ex.status, payload=ex.payload
```

Comment:
Nit. Though the keyword is optional, personally when the names don't match I think there's merit in explicitly defining them, i.e., `errors=[ex.error]`, which helps with readability.

Reply:
done